app/javascript/packs/stake_accounts/pool_stats.js

```diff
@@ -53,24 +53,24 @@ var StakePoolStats = Vue.component('StakePoolStats', {
               <strong class="text-purple">{{ (total_stake / 1000000000).toLocaleString('en-US', {maximumFractionDigits: 0}) }} SOL</strong>
             </div>
           </div>
-          
+
           <div class="col-md-4 pl-lg-4 pl-xl-5">
             <div class="stat-title-3 mb-2">
               <i class="fas fa-hand-holding-usd text-success mr-2"></i>Fees
-            </div> 
+            </div>
             <div>
               <span class="text-muted">Manager Fee:&nbsp;</span>
-              <strong class="text-success">{{ pool.manager_fee ? pool.manager_fee + '%' : 'N / A' }}</strong>
+              <strong class="text-success">{{ pool.manager_fee ? pool.manager_fee + '%' : 0 }}</strong>
             </div>
             <div class="mb-4">
               <span class="text-muted">Avg Commission:&nbsp;</span>
-              <strong class="text-success">{{ pool.average_validators_commission ? pool.average_validators_commission.toFixed(2) : 'N / A' }}%</strong>
+              <strong class="text-success">{{ pool.average_validators_commission ? pool.average_validators_commission.toFixed(2) : 0 }}%</strong>
             </div>
             <div class="mb-3">
               <span class="stat-title-3">
                 <i class="fas fa-chart-line text-purple mr-2"></i>APY:&nbsp;
               </span>
-              <strong class="text-purple">{{ pool.average_apy ? pool.average_apy.toFixed(2) + '%' : 'N / A' }}</strong>
+              <strong class="text-purple">{{ pool.average_apy ? pool.average_apy.toFixed(2) + '%' : 0 }}</strong>
```

Comment:
I think we can keep N/A in this single case. It's extremely unlikely that APY would be 0.